EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,352 @@
+---
```

Comment:
```suggestion
---
eip: 7809
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

Reply:
Thank you so much @abcoathup!